app/models/services/pubmed.rb

```diff
@@ -4,6 +4,8 @@ def initialize(options={})
     @search_url = @base_url + "/entrez/eutils/esearch.fcgi"
     @metadata_url = @base_url + "/entrez/eutils/esummary.fcgi"
     @abstract_url = @base_url + "/entrez/eutils/efetch.fcgi"
+    @pubmed_url = 'https://www.ncbi.nlm.nih.gov/pubmed/'
+    @pubmed_xml_options = '?report=xml&format=text'
```

Comment:
Instead of this, try using the `generate_uri` method. If you put these key/value pairs into the hash at the end of method it will add the parameters. Look at other uses of `#generate_uri` in this file.